lib/govuk_index/presenters/expanded_links_presenter.rb

```diff
@@ -4,60 +4,68 @@ def initialize(expanded_links)
       @expanded_links = expanded_links || {}
     end
 
-    def present
-      {
-        mainstream_browse_pages: mainstream_browse_page_slugs,
-        mainstream_browse_page_content_ids: content_ids("mainstream_browse_pages"),
-        organisations: organisation_slugs("organisations"),
-        organisation_content_ids: content_ids("organisations"),
-        part_of_taxonomy_tree: taxonomy_tree,
-        primary_publishing_organisation: organisation_slugs("primary_publishing_organisation"),
-        specialist_sectors: specialist_sector_slugs,
-        taxons: content_ids("taxons"),
-        topic_content_ids: content_ids("topics"),
-      }
+    def mainstream_browse_page_content_ids
+      content_ids("mainstream_browse_pages")
     end
 
-  private
-
-    attr_reader :expanded_links
+    def organisations
+      organisation_slugs("organisations")
+    end
 
-    def expanded_links_item(key)
-      expanded_links[key] || {}
+    def organisation_content_ids
+      content_ids("organisations")
     end
 
-    def values_from_collection(collection, key)
-      collection.map { |item| item[key] }
+    def primary_publishing_organisation
+      organisation_slugs("primary_publishing_organisation")
```

Comment:
This looks like it's going to return array for a single organisation. Is that OK?